js/App/AddMemberEvent.js

```diff
@@ -0,0 +1,92 @@
+import {
+  $todoAppContainer,
+  $todoListItem,
+  $appMemberAdd,
+} from "../content/shape.js";
+import { addMember, responseMemberApi } from "../server/AppServer.js";
+import {
+  countContainer,
+  initFilterEventListeners,
+} from "./todoAppCountContainer.js";
+
+import {
+  clickEraseButton,
+  clickInput,
+  clickCheckboxButton,
+  clickLabel,
+  checkPriority,
+} from "./mainLiItem.js";
+
+//member add  관련
+function makeAddList(value) {
+  value.innerHTML = $appMemberAdd;
+}
+
+function addMemberEvent(teamId) {
+  const $addTeamButton = document.querySelector("#add-user-button");
+  $addTeamButton.addEventListener("click", () => addMemberEventHandler(teamId));
+}
+
+async function addMemberEventHandler(teamId) {
+  const result = prompt("팀 이름을 입력해주세요");
+  if (/[\S]/gi.test(result) && result !== null) {
+    addMember(result, teamId);
+  } else alert("공백 혹은 정상적이지 않은 팀 이름 입력");
+}
+
+function memberRender() {
+  const $domTeamListContainer = document.querySelector(
+    ".todoapp-list-container"
+  );
+  const length = $domTeamListContainer.children.length;
+  for (let i = 0; i < length; i++) {
+    $domTeamListContainer.children[0].remove();
+  }
```

Comment:
```suggestion
$domTeamListContainer.innerHTML = '원하는 초기값'
```

혹시 위와 같이 작성하지 않고 for문으로 순회하면서 자식 노드를 지우신 이유가 있을까요?